src/js/src/components/ToolboxComponents/NgramNetarchive.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <div class="linkGraphContainer">
+    <h2 class="toolboxHeadline">
+      Ngram Netarchive
+    </h2>
+    <div class="contentContainer">
+      <transition name="loading-overlay">
+        <loading-overlay />
+      </transition>
+      <notifications />
```

Comment:
Why are we needing the notifications here again? Just curious...

Reply:
You are on 🔥 today! Leftover from the refactor in toolbox we can rely on the global one:)